pvlib/iotools/pvgis.py

```diff
@@ -126,6 +126,21 @@ def get_pvgis_hourly(latitude, longitude, start=None, end=None,
     metadata : dict
         Dictionary containing metadata
 
+    Raises
+    ------
+    requests.HTTPError
+        If the request response status is ``HTTP/1.1 400 BAD REQUEST``, then
+        the error message in the response will be raised as an exception,
+        otherwise raise whatever ``HTTP/1.1`` error occurred
+
+!!! info "Radiation databases and timestamp convention"
```

Comment:
I don't recognize this `!!!` syntax -- is it valid RST?  The docs aren't rendering correctly, maybe because of this line.  I'd have expected to see something like this:

```
    .. warning:: PVGIS provides access to a number of different
        solar radiation...
```

https://pvlib-python--1186.org.readthedocs.build/en/1186/generated/pvlib.iotools.get_pvgis_hourly.html

Reply:
@kanderso-nrel I have added a section called "Hint" instead. The PR is now complete and can be merged if there are no other comments.

Reply:
@kanderso-nrel I have added a section called "Hint" instead. The PR is now complete and can be merged if there are no other comments.

Reply:
@kanderso-nrel I have added a section called "Hint" instead. The PR is now complete and can be merged if there are no other comments.